padle/padle.py

```diff
@@ -0,0 +1,801 @@
+    #     final = []
+    #     # await ctx.send(mgraph.max_monster_id)
+    #     for i in range(0, 8800): # hard coded bc max_monster_id as calculated above is 15k???
+    #         monster = await dbcog.find_monster(str(i), ctx.author)
+    #         try:
+    #             nextTrans = mgraph.get_next_transform(monster)
+    #             prevTrans = mgraph.get_prev_transform(monster)
+    #             if(monster is not None and monster.name_en is not None and monster.on_na and
+    #                 monster.series.series_type != 'collab' and
+    #                 ((monster.sell_mp >= 50000 and (monster.superawakening_count > 1 or prevTrans is not None)) or
+    #                  "Super Reincarnated" in monster.name_en) and not monster.is_equip and
+    #                     nextTrans is None and monster.level >= 99):
+    #                 final.append(str(i))
+    #         except Exception as e:
+    #             print("Error " + str(i))
+    #             pass
+    #     await ctx.tick()
+    #     print(",".join(final))
+
+    async def ensure_active_game(self, ctx):
+        prefix = ctx.prefix
+        if ctx.guild is not None and not await self.can_play_in_guild(ctx):
+            await ctx.send("You can only play PADle in DMs!")
+            return True
+        if not await self.config.user(ctx.author).start():
+            await ctx.send(f"You have not started the game of PADle yet, try `{prefix}padle start`!")
+            return True
+        if await self.config.user(ctx.author).done():
+            await ctx.send("You have already played today's PADle!")
+            return True
+        return False
+
+    @padle.command()
+    async def guess(self, ctx, *, guess):
+        """Guess a card for the daily PADle"""
+        if await self.ensure_active_game(ctx):
+            return
+        dbcog = await self.get_dbcog()
+
+        guess_monster = await dbcog.find_monster(guess, ctx.author.id)
+        if guess_monster is None:
+            close_menu = ClosableEmbedMenu.menu()
+            props = PADleMonsterConfirmationViewProps("Monster not found, please try again.")
+            query_settings = await QuerySettings.extract_raw(ctx.author, self.bot, guess)
+            state = ClosableEmbedViewState(ctx.author.id, ClosableEmbedMenu.MENU_TYPE, guess,
+                                           query_settings, PADleMonsterConfirmationView.VIEW_TYPE, props)
+            await close_menu.create(ctx, state)
+            return
+
+        m_embed = EmbedView(
+            EmbedMain(
+                title=MonsterHeader.menu_title(guess_monster).to_markdown(),
+                description="Did you mean this monster?"),
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(guess_monster.monster_id))).to_embed()
+        did_you_mean_msg = await ctx.send(embed=m_embed)
+        confirmation = await get_user_reaction(ctx, did_you_mean_msg, YES_EMOJI, NO_EMOJI, timeout=20)
```

Comment:
I added this to the TSUtils one.  You can put an embed in for the message argument, and it will work now